src/util/RoundRobinLB.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AbstractLoadBalancer} from './AbstractLoadBalancer';
+import {randomInt} from '../Util';
+import {Member} from '../core/Member';
+
+// tslint:disable-next-line:no-bitwise
+const INITIAL_SEED_CAP = Math.ceil(Number.MAX_SAFE_INTEGER / (1 << 16));
```

Comment:
Are there any reasons not to use `1 << 16` instead of `Math.ceil(Number.MAX_SAFE_INTEGER / (1 << 16))` as the cap? 65K sounds like a safe value for the cap to me. WDYT?

Reply:
I thought you meant set the divisor to (1 << 16) :smile:.  My bad